streams/writable-streams/write.js

```diff
@@ -164,6 +164,30 @@ promise_test(t => {
       .then(() => promise_rejects(t, new TypeError(), writer.close(), 'close() should be rejected'));
 }, 'when sink\'s write throws an error, the stream should become errored and the promise should reject');
 
+promise_test(t => {
+  const passedError = new Error('horrible things');
```

Comment:
Thanks. Removed passedError and thrownError from all the tests in this file and switched to use error1/error2 in a48c302e5a6af51f80c0a98439379af427411444